### DISCUSSION TITLE
Connecting to PicoCNC over WiFi

### DISCUSSION THREAD

**Discussion by gilsegev** [phil-barrett/PicoCNC#15] [General]
I am thinking about purchasing the PicoCNC as a replacement for an Arduino based CNC router build. I like to idea of using WiFi for sending GCODE to the controller but couldn't find much info on how this is actually done. Assuming I buy the Pico W, how is the connection actually made over WiFi? 

Thank you!

**Comment by phil-barrett** [comment]
It's fairly easy to do.  Page 16 of the [User Manual](https://github.com/phil-barrett/PicoCNC/blob/main/PicoCNC%20User%20Manual.pdf) has the instructions.  Note that you have to choose between WiFi or Ethernet - you can't use both.
